pom.xml

```diff
@@ -86,7 +86,7 @@
     <google.cloud.storage.version>2.3.0</google.cloud.storage.version>
     <google.cloud.datastore.version>1.105.1</google.cloud.datastore.version>
     <google.protobuf.java.version>3.19.4</google.protobuf.java.version>
-    <google.tink.version>1.3.0-rc3</google.tink.version>
+    <google.tink.version>1.5.0</google.tink.version>
```

Comment:
why this change?

Reply:
I encountered dependency errors while running mvn clean install after resolving all the checkstyle issues. The build was failing due to the outdated version of google.tink.version. Updating it to 1.5.0 resolved the issue, and no other changes were made.

Reply:
please do not make dependency changes in the e2e tests PR.

Reply:
Refactor CdfPluginPropertyLocator alignment and update build dependencies.
-Fixed indentation alignment for partitionFrom, partitionTo, filter and Output Schema.
-Updated pom.xml as it is.